webpack.examples.config.js

```diff
@@ -1,11 +1,11 @@
 const commonConfig = require('./webpack.common.config.js');
 
 commonConfig.entry = {
-  userChip: './examples/userChip.js'
+  UserChip: './src/UserChip/UserChip.example.jsx'
 };
 
 commonConfig.output = {
-  filename: '[name].js',
+  filename: '[name]/[name].js',
```

Comment:
Does this also work for components that are nested more deeply, e.g. `ModuleType/ModuleA/ModuleA.jsx` or don't we have such a nested structure in the examples?

Reply:
Should work for the most cases. We might want to improve this later.